application/src/main/resources/thingsboard.yml

```diff
@@ -65,6 +65,7 @@ server:
       max_per_user: "${TB_SERVER_WS_DYNAMIC_PAGE_LINK_MAX_PER_USER:10}"
     max_entities_per_data_subscription: "${TB_SERVER_WS_MAX_ENTITIES_PER_DATA_SUBSCRIPTION:10000}"
     max_entities_per_alarm_subscription: "${TB_SERVER_WS_MAX_ENTITIES_PER_ALARM_SUBSCRIPTION:10000}"
+    default_queue_messages_per_session: "${TB_SERVER_WS_DEFAULT_QUEUE_MESSAGES_PER_SESSION:1000}"
```

Comment:
1) use 
```max_queue_messages_per_session```
and validate the device profile according to this limit on save or update;
2) 0 on UI means ```max_queue_messages_per_session```;
3) The text field label should be: "Maximum size of the message queue per session."
4) The text hint should be: "The size of the queue is also limited by the system configuration."